packages/ember-testing/tests/helpers_test.js

```diff
@@ -414,10 +398,10 @@ test('`click` triggers appropriate events in order', function() {
     events = [];
     return click('.index-view input[type=checkbox]');
   }).then(function() {
-    // i.e. mousedown, mouseup, change:true, click, click:true
+    // i.e. mousedown, mouseup, click
     // Firefox differs so we can't assert the exact ordering here.
     // See https://bugzilla.mozilla.org/show_bug.cgi?id=843554.
-    equal(events.length, 5, 'fires click and change on checkboxes');
+    equal(events.length, 3, 'fires click and change on checkboxes');
```

Comment:
They were just above in the setup: https://github.com/emberjs/ember.js/pull/13617/files#diff-f834632dddeade3002a271a3e9fb68a3L369. Can you tweak this one to register a custom `{{x-checkbox` helper so that we can still confirm these events?